nixos/modules/services/mail/postfix.nix

```diff
@@ -199,27 +327,29 @@ in
         default = true;
         description = "Whether to enable smtp in master.cf.";
       };
-      
+
       enableSubmission = mkOption {
         type = types.bool;
         default = false;
-        description = "Whether to enable smtp submission";
+        description = "Whether to enable smtp submission.";
```

Comment:
I just have the impression that the postfix documentation on this is going to be less vague than this.